app/src/components/StyledButton/StyledButton.module.css

```diff
@@ -0,0 +1,267 @@
+.typeA {
```

Comment:
More descriptive descriptions here maybe?

Are the buttons used in specific situtations?